pkg/sysregistries/system_registries_test.go

```diff
@@ -1,21 +1,22 @@
 package sysregistries
 
 import (
+	"github.com/containers/image/types"
 	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
 var testConfig = []byte("")
 
 func init() {
-	readConf = func() ([]byte, error) {
+	readConf = func(_ *types.SystemContext) ([]byte, error) {
```

Comment:
BTW the tests could now use fixture files and use `SystemContext` to point to them, instead of having the `readConf` variable and overriding the implementation.

Not at all a blocker for this PR, only a newly available option.

Reply:
:+1: to having this indeed

Reply:
Indeed, I thought about that too, but will leave alone for now.  If I change the tests in the future, I'll likely move to that approach.